Add OpenIdfa files to framework target

lisional opened the pull request:
Hello,

We are using the universal framework generated by the target "SnowplowTracker-iOS-Static".
When building our app we encountered a linking error:
```
Undefined symbols for architecture arm64: "_OBJC_CLASS_$_OpenIDFA", referenced from: 
objc-class-ref in SnowplowTracker(SPUtilities.o)
ld: symbol(s) not found for architecture arm64
```

OpenIdfa files are not included in the SnowplowTracker-iOS-Static target, meaning we have to add them to our codebase manually to make linking succeed. Preprocessor macro **SNOWPLOW_NO_OPENIDFA** is useless in this case since we are using the compiled framework.

We have a workaround, but it would be great to not have to include this files in our codebase.

Thanks

lisional commented:
Hello,
Thanks for your quick answer, is this fix not enough? Or are you talking about the code coverage decrease?

Carthage support would help us for sure, but the present fix could help others not using Carthage/Pods.

mhadam commented:
Yup this fix is fine! I'm staging it into the release/1.0.0 branch.

lisional commented:
I have tested it locally, all good for our needs. I was able to generate the framework, drop it in our project, delete our OpenIdfa files and compile successfully our app.

Small issue with Xcode 10 by the way; I think that the python script used for assembling the framework  binary (final Build Phase) does not work with the new build system. I had to switch back to the legacy one.

I tried fixing this as well, but did not find the cause of the issue (and had not enough time to investigate too)

mhadam commented:
Ok thanks! If it's not a big issue, I'll leave the new build system in place.

lisional commented:
I need to amend what I said; release/1.0.0 is not working for me.
`dev/snowplow-objc-tracker/Snowplow/SPEventStore.m:27:9: 'FMDB/FMDB.h' file not found`

Even after running carthage bootstrap.
I thought it was successfull, but missed this error in xcode

mhadam commented:
And you added the dependency frameworks (Reachability and FMDB) from the Carthage build folder right?

lisional commented:
Yes, I added FMDB and Reachability to the linked librariries list. 
You have not updated the Framework search paths, Frameworks.xcodeproj has no way to know where to look FMDB and Reachability frameworks. I added it like this:
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/1579085/51709222-d99d7600-1ff3-11e9-9757-234f668d071b.png">

Please note I am using Frameworks.xcodeproj since Snowplow.workspace seems broken (no sheme found).
